tests/pytests/test_issues.py

```diff
@@ -127,3 +127,30 @@ def test_issue1834(env):
   conn.execute_command('HSET', 'doc', 't', 'hell hello')
 
   env.expect('FT.SEARCH', 'idx', 'hell|hello', 'HIGHLIGHT').equal([1L, 'doc', ['t', '<b>hell</b> <b>hello</b>']])
+
+def test_issue1880(env):
+  # order of iterator in intersect is optimized by function
+  env.skipOnCluster()
+  conn = getConnectionByEnv(env)
+  env.cmd('FT.CONFIG', 'SET', '_PRINT_PROFILE_CLOCK', 'false')
+  conn.execute_command('FT.CREATE', 'idx', 'SCHEMA', 't', 'TEXT')
+  conn.execute_command('HSET', 'doc1', 't', 'hello world')
+  conn.execute_command('HSET', 'doc2', 't', 'hello')
+
+  excepted_res = [[1L, 'doc1', ['t', 'hello world']], 
```

Comment:
Not a change request, but this is exactly when JSON output of ft.profile will makes sense, so we don't need to rely on parsing user output.